blazor/datagrid/paging.md

```diff
@@ -7,443 +7,1162 @@ control: DataGrid
+        await Grid.Refresh();
+    }
+
+    private async Task InputHandler(InputEventArgs args)
+    {
+        if (args.Value != string.Empty)
+        {
+            value = int.Parse(args.Value);
+        }
     }
 }
-```
+{% endhighlight %}
+{% highlight c# tabtitle="OrderData.cs" %}
+public class OrderData
+    {
+        public static List<OrderData> Orders = new List<OrderData>();
+
+        public OrderData()
+        {
+
+        }
+        public OrderData(int? OrderID,string CustomerID,string ShipCity, string ShipName)
+        {
+            this.OrderID = OrderID;    
+            this.CustomerID = CustomerID;
+            this.ShipCity = ShipCity;
+            this.ShipName = ShipName;           
+        }
+
+        public static List<OrderData> GetAllRecords()
+        {
+            if (Orders.Count() == 0)
+            {
+               
+                int OrderID = 10248;
+
+                for (int i = 1; i < 7; i++)
+                {
+                    Orders.Add(new OrderData(OrderID+1, "VINET", "Reims", "Vins et alcools Chevali"));
+                    Orders.Add(new OrderData(OrderID+2, "TOMSP", "Münster", "Toms Spezialitäten"));
+                    Orders.Add(new OrderData(OrderID+3, "HANAR", "Rio de Janeiro", "Hanari Carnes"));
+                    Orders.Add(new OrderData(OrderID+4, "VICTE", "Lyon", "Victuailles en stock"));
+                    Orders.Add(new OrderData(OrderID+5, "SUPRD", "Charleroi", "Suprêmes délices"));
+                    Orders.Add(new OrderData(OrderID+6, "HANAR", "Lyon", "Hanari Carnes"));
+                    Orders.Add(new OrderData(OrderID+7, "CHOPS", "Rio de Janeiro", "Chop-suey Chinese"));
+                    Orders.Add(new OrderData(OrderID+8, "RICSU", "Münster", "Richter Supermarkt"));
+                    Orders.Add(new OrderData(OrderID+9, "WELLI", "Reims", "Wellington Import"));
+                   
+                    OrderID += 9;
+                }
+            }
+            return Orders;
+        }
 
-N> You can refer to our [Blazor Grid Paging](https://www.syncfusion.com/blazor-components/blazor-datagrid/paging) Feature tour page to know about paging and its feature representations.
-<br/> You can refer to our [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore our [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
+        public int? OrderID { get; set; }
+        public string CustomerID { get; set; }
+        public string ShipCity { get; set; }
+        public string ShipName { get; set; }
+   }
+{% endhighlight %}
+{% endtabs %}
 
-## Display All records through pager dropdown
+{% previewsample "https://blazorplayground.syncfusion.com/embed/htrTXCZuLdnPIeht?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
```

Comment:
change SftextBox to SfNumericTextBox without arrow icons

Reply:
Resolved